03.bowling/frame.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class Frame
+  attr_reader :shots
+
+  def initialize(shots, is_last_frame: false)
+    @shots = shots
+    @is_last_frame = is_last_frame
+  end
+
+  def pins
+    @shots.map(&:pin)
+  end
+
+  def score(index, all_frames)
+    base_score = calc_base_score
+    base_score + calculate_bonus(index, all_frames)
```

Comment:
```suggestion
    calc_base_score + calculate_bonus(index, all_frames)
```

まとめちゃってもいいかも。
あと、calcとcalculateで表記が揺れてるので合わせた方がいいですね